config.json

```diff
@@ -875,6 +875,19 @@
           "loops",
           "pointers"
         ]
+      },
+      {
+        "slug": "high-scores",
+        "name": "High Scores",
+        "uuid": "10448fb8-60eb-4c67-93e8-488ecbde8676",
+        "practices": [
+          "vector-arrays"
+        ],
```

Comment:
perhaps "class", too?

Reply:
I have not yet finished the syllabus to a working degree yet, so I want to postpone these entries.